hw2/05-form.html

```diff
@@ -10,10 +10,57 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="05-form.js"></script>
   </head>
-  <body>
-    <main>
+  <body class="bg-primary container mt-5">
+    <main class=" bg-light rounded p-4 d-flex flex-column max-sm-75">
       <form>
-        <h1>Exercise 05 - Form</h1>
+        <div class="mb-3">
+          <label for="fullName" class="form-label">Full name</label>
+          <input type="text" name="full-name" class="form-control" id="fullName">
+        </div>
+        <div class="mb-3">
+          <label for="email-input" class="form-label">Email address</label>
+          <input type="email" name="email" class="form-control" id="email-input">
+        </div>
+        <div class="mb-3">
+          <label for="registration-status" class="form-label">Registration Status</label>
+          <select class="form-select" name="registration-status" id="registration-status" aria-label="Default select example">
+            <option selected>Choose an Option</option>
+            <option value="1">Registered</option>
+            <option value="2">Unregistered</option>
+            <option value="3">Waitlist</option>
+          </select>
+        </div>
+        <div class="form-group">
+          <legend>Have you taken any of the following courses?</legend>
```

Comment:
You could use a label here instead of a legend to properly label your form check.